### DISCUSSION TITLE
Are private repos deleted when I downgrade to a free plan?

### DISCUSSION THREAD

**Discussion by hellocakebread** [community/community#23328] [New to GitHub]
<p>I’m not using my private repos for the next few months, so I’m thinking of downgrading to free. Does anyone know if those repos will still exist so that when I’m ready to start work again I can resume payments and get my repos back? Thanks!</p>

**Comment by nadiajoyce** [comment, marked as answer]
<p>Hi <a class="mention" href="/hellocakebread">@hellocakebread</a> Great question! If you downgrade to free, your private repos will actually just lock (meaning they’ll be inaccessible to you, but they <em>won’t</em> get deleted). If you ever need access to those private repos again, you can just re-upgrade to a paid account and they’ll be there waiting for you, just as you left them.</p>
<p>We do often recommend, though, that before you downgrade, you make sure you have a copy of any private repositories you’d like to keep. That way, you can still have local access, at least, to that content while on the free plan.</p>
<p>Let me know if you have any other questions or if I can help with anything else!</p>

**Comment by hellocakebread** [comment]
<p>Great, thank you!</p>

**Comment by danielmee** [comment]
<p>My question exactly - it would be great if Github could put this info into their help section or in the little explanation on the settings page as it’s not clearly explained anywhere else.</p>

**Comment by angela-crist** [comment]
<p>Hi <a class="mention" href="/danielmee">@danielmee</a>,</p>
<p>Thanks for this feedback! We’re always working to improve GitHub and the GitHub Community Forum, and we consider every suggestion we receive. I’ve logged your feature request in our internal feature request list. Though I can’t guarantee anything or share a timeline for this, I can tell you that it’s been shared with the appropriate teams for consideration.</p>
<p>Please let me know if you have any other questions.</p>
<p>Cheers!</p>

**Reply by nbro** [reply]
Seriously? How much does it take to write a sentence that point this information out? You just need to copy and paste the answer above.

**Comment by patchthecode** [comment]
<p>Is the answers here still relevant since github now offers private repositories for free?</p>

**Comment by berndtlindner** [comment]
<p>Very good question, I downgraded from Pro to Free and expected to still have my “unlimited” private repos.</p>
<p>The only difference being I may only now have 3 collaborators in my private repos (see <a href="https://github.com/pricing" rel="nofollow noopener">https://github.com/pricing</a>) But now I can’t find my private repo on my profile?</p>
<p>I sure hope they are not “locked” or worse deleted</p>

**Comment by sudhirpoodar** [comment]
<p>Hi Team,</p>
<p>Currently we are using approx 100 private repositories.<br>
As per Github new update here ‘’ <a href="https://github.blog/2020-04-14-github-is-now-free-for-teams/" rel="nofollow noopener">https://github.blog/2020-04-14-github-is-now-free-for-teams/</a> ‘’</p>
<p>can you check and reply on 2 query.</p>
<ol>
<li>If we downgrade our plan to Free Plan then what services and features is not available in free plan as compare to Platinum plan? </li>
<li>can we upgrade to Platinum plan from free plan in future?</li>
</ol>
